content/pt/docs/tutorials/kubernetes-basics/scale/scale-intro.html

```diff
@@ -0,0 +1,121 @@
+            <div class="col-md-8">
+                <div id="myCarousel" class="carousel" data-ride="carousel" data-interval="3000">
+                    <ol class="carousel-indicators">
+                        <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+                        <li data-target="#myCarousel" data-slide-to="1"></li>
+                    </ol>
+                      <div class="carousel-inner" role="listbox">
+                        <div class="item carousel-item active">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling1.svg">
+                        </div>
+
+                        <div class="item carousel-item">
+                          <img src="/docs/tutorials/kubernetes-basics/public/images/module_05_scaling2.svg">
+                        </div>
+                      </div>
+
+                      <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                        <span class="sr-only ">Anterior</span>
+                      </a>
+                      <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                        <span class="sr-only">Próximo</span>
+                      </a>
+
+                    </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+
+                <p>Escalar um Deployment garantirá que novos Pods serão criados e agendados para nós de processamento com recursos disponíveis. O escalonamento aumentará o número de Pods para o novo estado desejado. O Kubernetes também suporta o auto-escalonamento (<a href="/docs/user-guide/horizontal-pod-autoscaling/">autoscaling</a>) de Pods, mas isso está fora do escopo deste tutorial. Escalar para zero também é possível, e isso terminará todos os Pods do Deployment especificado.</p>
+
+                <p>Executar múltiplas instâncias de uma aplicação irá requerer uma forma de distribuir o tráfego entre todas elas. Serviços possuem um balanceador de carga integrado que distribuirá o tráfego de rede entre todos os Pods de um Deployment exposto. Serviços irão monitorar continuamente os Pods em execução usando endpoints para garantir que o tráfego seja enviado apenas para Pods disponíveis.</p>
+
+            </div>
+            <div class="col-md-4">
+                <div class="content__box content__box_fill">
+                    <p><i>O Escalonamento é obtido pela mudança do número de réplicas em um Deployment.</i></p>
+                </div>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="row">
+            <div class="col-md-8">
+                <p>No momento em que tiver múltiplas instâncias de uma aplicação em execução, será capaz de fazer <i>Rolling updates</i> sem indisponibilidade. Nós cobriremos isso no próximo módulo. Agora, vamos ao terminal online e escalar nossa aplicação.</p>
```

Comment:
We can use the glossary for this term: "Rolling updates".

Reply:
Hey @jcjesus! Would you mind proposing a translation for "Rolling updates"? I haven't found any good meaning in Portuguese for it. I saw that the translation for some language (Deustch, Polski) for this page kept this term without translation. I would appreciate your suggestion.

Reply:
Could be "Atualizações Contínuas", please, make double-check if adherent in this context.
Thanks

Reply:
@jcjesus I liked your translation for it, even found another: "Atualização gradual" that would fit perfectly. It refers to the action of replacing some outdated pods for updated pods when a deployment with a new image is configured.